nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-audit/src/test/resources/es-audit-pageprovider-test-contrib.xml

```diff
@@ -3,94 +3,94 @@
 <component name="org.nuxeo.elasticsearch.PageProviderservice.test.contrib">
 
   <extension target="org.nuxeo.ecm.platform.query.api.PageProviderService"
-    point="providers">
+             point="providers">
 
     <genericPageProvider name="SimpleESAuditPP"
-      class="org.nuxeo.elasticsearch.audit.pageprovider.ESAuditPageProvider">
-       <pattern>
-       {
-				 "query": {
-           "bool": {
-             "must": {
-               "match": {
-                  "docPath": {
-                    "query": "/dummy"
+                         class="org.nuxeo.elasticsearch.audit.pageprovider.ESAuditPageProvider">
+      <pattern>
+        {
+          "query":{
+            "bool":{
+              "must":{
+                "match":{
+                  "docPath":{
+                    "query":"\/dummy"
```

Comment:
Looks like it is more a rewriting than just a cleaning/format?

Reply:
you right i used a JSON plugin formatter and i didn't pay more attention. thx